tests/rptest/tests/random_node_operations_test.py

```diff
@@ -105,7 +105,7 @@ def _setup_test_scale(self, num_to_upgrade):
         else:
             # container test setup
             if num_to_upgrade > 0:
-                self.should_skip = True
+                self.should_skip = False
```

Comment:
Was this intended to be committed?

Reply:
no, actually not, sorry i was distracted by something else